Irina/src/main/java/FirstTasks/DisplayNumberUsingFor.java

```diff
@@ -0,0 +1,12 @@
+package FirstTasks;
+/*
+Display numbers from 1 to 10 using for()
+ */
+
+public class DisplayNumberUsingFor {
+    public static void main(String[] args) {
+        for (int number =1;number<=10;number++){
+            System.out.println(number);
+        }
+    }
+}
```

Comment:
Use Ctrl + Alt + L for code formating. Do this for all classes

Reply:
Done